pkg/profiles/openapi.go

```diff
@@ -71,31 +73,31 @@ func swaggerToServiceProfile(swagger spec.Swagger, namespace, name, clusterDomai
 		path := path.Join(swagger.BasePath, relPath)
 		pathRegex := pathToRegex(path)
 		if item.Delete != nil {
-			spec := mkRouteSpec(path, pathRegex, http.MethodDelete, item.Delete.Responses)
+			spec := mkRouteSpec(path, pathRegex, http.MethodGet, item.Delete.Responses, item.Delete)
```

Comment:
This should still be `http.MethodDelete` right?

Reply:
@Pothulapati Yes, you're right. I'll correct that.